chart/templates/deployment.yaml

```diff
@@ -109,6 +109,10 @@ spec:
 {{ toYaml .Values.resources | indent 10 }}
         volumeMounts:
 {{- if .Values.additionalTrustedCAs }}
+        - mountPath: /etc/ssl/certs/ca-additional.pem
```

Comment:
I'm confused here, didn't we have a commit a while back to override the system ssl directory? Running a shell on rancher/rancher:v2.2.1 shows `SSL_CERT_DIR=/etc/rancher/ssl`

This new change will have no affect as long as SSL_CERT_DIR is set.

Reply:
`SSL_CERT_DIR` is for Golang based, `/etc/ssl/certs` is used by the Git binary. See the validation step in https://github.com/rancher/rancher/issues/19473#issuecomment-481915643

Reply:
Ok, I understand. Probably shouldn't have tried to change that SSL_CERT_DIR in the first place :(